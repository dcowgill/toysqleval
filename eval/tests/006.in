create table test (
    n integer,
    f number,
    s varchar
);

insert into test (f,s,n) values (1.1, 'hello', 11);
insert into test (n,s,f) values (7, 'world', 2.2);
insert into test (f) values (3.3);
insert into test (n) values (13);

select sum(f) from test;
select sum(n) from test;
select count(f), count(n), count(s), count(*) from test;
select (sum(f)*sum(n))/2, sum(f*n)/2, sum(f)/count(f) from test;
