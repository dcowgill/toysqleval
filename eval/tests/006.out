OK
OK
OK
OK
OK
 ?  
-----
 6.6
 ? 
----
 31
 ? | ? | ? | ?
---+---+---+---
 3 | 3 | 2 | 4
 ?     | ?                  | ?                 
-------+--------------------+--------------------
 102.3 | 13.750000000000002 | 2.1999999999999997
